sale_stock_picking_blocking/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Stock Picking Blocking",
+    "summary": "Allow you to block the creation of deliveries from a sale order.",
+    "version": "17.0.1.0.0",
+    "author": "Eficent, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Sales",
+    "depends": ["sale_stock"],
```

Comment:
I suggest that we include `sale_management` in depended, since the only way users used this module on UI only when they can see sale menu, [this one ](https://github.com/odoo/odoo/blob/17.0/addons/sale_management/views/sale_management_menus.xml#L5)activate the sale menu